guides/v2.0/extension-dev-guide/validate/test-module.md

```diff
@@ -16,17 +16,18 @@ redirect_from:
 
```

Comment:
Thanks! Everything looks good now. We’ll have some more feedback for you soon.

From: Vinai Kopp [mailto:notifications@github.com]
Sent: Wednesday, March 30, 2016 5:16 PM
To: magento/devdocs
Cc: Johnson, Steven
Subject: Re: [magento/devdocs] Document running PHP unit tests (#586)


In guides/v2.0/extension-dev-guide/validate/test-module.md<https://github.com/magento/devdocs/pull/586#discussion_r57973348>:

> @@ -16,17 +16,18 @@ redirect_from:

>

Removed

—
You are receiving this because you commented.
Reply to this email directly or view it on GitHub<https://github.com/magento/devdocs/pull/586/files/166e7e7fa4d446fdefb185aa83322f06c13ac81f#r57973348>